PinpointKit/PinpointKit/Sources/Configuration.swift

```diff
@@ -60,7 +60,7 @@ public struct Configuration {
                 feedbackCollector: FeedbackCollector = FeedbackNavigationController(),
                 editor: Editor = EditImageViewController(),
                 sender: Sender = MailSender(),
-                feedbackRecipients: [String]? = nil) {
+                feedbackRecipients: [String]) {
```

Comment:
Yeah, I meant the stored property on the collector, but I see your point. Let's keep it as-is.